### ISSUE TITLE
A11y_SQLAzureDataStudio_SchemaCompare_MenuBar_ScreenReader: Status message not announced for updating content after selecting any button from menu bar.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9399]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#MAS4.1.3;#Benchmark; #A11y_SQLAzureDataStudioBenchmark;#A11ySev2;#NVDA;#JAWS;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS reference: MAS4.1.3
Screen Reader:NVDA(2019.3.1),JAWS

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Schema Compare Menu bar below Schema Compare tab.
11. Check if status message is announced for all the updations happening on the screen.

**Actual:**
Status message is not announced for the updations that happen after pressing enter on the any of the menu option present in the menu bar.

Issue observed in NVDA and JAWS.
**Expected:**
Status message should be announced for the content that is updated on the screen by the screen reader.

**User Impact:**
Screen reader users will not know that what is the content that is getting updated on the screen.

**Attachment link reference:**
[9399_A11y_SQLAzureDataStudio_Extension_SchemaCompare_MenuBar_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291412/9399_A11y_SQLAzureDataStudio_Extension_SchemaCompare_MenuBar_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
for the long running actions, we need to see why loading component is not working (it should read loading/loading completed)
for the actions that finish fast, i am not sure whether we need to announce anything.

**Comment by alanrenmsft** [comment]
for the actions that finish fast, I will ask the accessibility team for guideline

**Comment by Charles-Gagnon** [comment]
Discussed this in office hours with v-tayazd - for actions which execute immediately we shouldn't be announcing that an action is occurring as that tends to be too verbose.

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by Harika1702** [comment]
https://user-images.githubusercontent.com/100270074/218076588-2ed21a8f-4ef8-4090-b3b4-cddf891cc23f.mp4

Verified this issue in the below environment.
App name: Azure data studio
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
